semgrep_ocaml.yml

```diff
@@ -0,0 +1,210 @@
+rules:
```

Comment:
At some point I would rather like use --config https://semgrep.dev/r/ocaml but I could not find a way to create an ocaml ruleset, so I've created a policy and downloaded the corresponding yaml file.